packages/bpk-tokens/tokens/base.raw.ios.json

```diff
@@ -393,6 +393,62 @@
       "originalValue": "{!WHITE}",
       "name": "BACKGROUND_TERTIARY_LIGHT_COLOR"
     },
+    "BORDER_RADIUS_LG": {
```

Comment:
Same as my comment before around do we need them given they have corner radius

Reply:
Yes, otherwise every RN usage has to look like this:
`borderRadius: Platform.OS === 'iOS' ? cornerRadiusSm : borderRadiusSm`

And in RN `borderRadius` is a more 1st-class name. Corner radius is very Native iOS

Reply:
Sure thing! This is interesting as myself and Shaun had this convo in slack and somehow even though we use `borderRadius: borderRadiusSm` in code and have not done your suggestion it always worked even though we didn't define the `cornerRadiusSm` 🤯

Reply:
Yeah because `bpk-tokens` in the react-native codebase is 2 major versions behind 🙈